.github/workflows/build_and_test.yml

```diff
@@ -14,16 +14,22 @@ on:
   pull_request:
     paths-ignore:
       - "**.md"
-  
+
 jobs:
   build-and-test:
     name: Build and Test
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        include:
+          - os: "macos-latest"
+            # The MacOS runner doesn't have Docker installed, os can't run
+            # `dazel`. Fall back to `bazel` instead.
```

Comment:
I'm going to be a nit picker and ask to try and make this comment be the same (upper/lower case and all) as in eventuals please!

Reply:
Done